src/icons/README.md

```diff
@@ -32,13 +33,17 @@ installed by default. You do not need `bootstrap-icons` as a dependency.
       names &mdash; <code>arrow-up-down</code> renamed <code>arrow-down-up</code> and
       <code>people-circle</code> renamed <code>person-circle</code>.
     </li>
+    <li>
+      <strong>Alpha 5 changes:</strong> In addition to over 300 new icons, some icons have changed
+      names &mdash; <code>camera</code> renamed <code>camera2</code>.
+    </li>
   </ul>
 </div>
 
 ## Icons
 
-The library includes over 670 icons. Use the explorer below to search and browse the available
-icons.
+The library includes over {{ bootstrapIconsCount }} icons. Use the explorer below to search and
+browse the available icons.
 
 <div>
```

Comment:
Would it makes sense to add a scrollbar to the icons list?
Currently i feel like it can be a bit daunting, and takes a lot of scroll to get to the examples, and the list is only gonna get larger going forward.

Something like this.
`style="max-height: 750px; overflow-y: auto; padding: 15px;"`

Reply:
We thought about that before but I think we would be better of moving the "Icon explorer" section more to the bottom of the page.

Reply:
That could work too.

Reply:
Will look into that tomorrow.